src/projectBank2/Account.java

```diff
@@ -0,0 +1,77 @@
+package projectBank2;
+
+import java.math.BigDecimal;
+
+public class Account {
+
+    private BigDecimal myPLN;
+    private BigDecimal myGBP;
+    private BigDecimal myUSD;
+    private String kontoPLN;
+    private String kontoGBP;
+    private String kontoPLNSecond;
+
+    public Account() {
+        kontoPLN = "Account number: 123456";
+        kontoGBP = "Account number: 654321";

```

Comment:
Also if you need "Account number:" text, it should not be part of number itself. Make it account only a number, if you need to display such text, do it in Account provider. E.g:
```java
Account account = new Account();
account.setAccountNumber("1234");
System.out.println("Account number: " + account.getAccountNumber());
```